build.gradle

```diff
@@ -16,13 +41,10 @@ java {
   }
 }
 
-repositories {
-    mavenCentral()
-}
-
 dependencies {
-    testImplementation 'org.junit.jupiter:junit-jupiter:5.10.3'
-    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.10.3'
+    testImplementation(platform('org.junit:junit-bom:5.10.3'))
+    testImplementation('org.junit.jupiter:junit-jupiter')
```

Comment:
Did you omit a specific version on purpose here (and in the line below)?

Reply:
Yes, the versions are given by the platform defined via the bom. This ensures that the versions are aligned i.e. also with the impacted-test-engine dependencies

Reply:
Ah, haven't payed close enough attention to notice `platform` and didn't know the concept of bom yet. Thanks for the explanation! :)